packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -56,9 +56,8 @@ abstract class UnpackMacOS extends Target {
     final Directory targetDirectory = environment
       .outputDir
       .childDirectory('FlutterMacOS.framework');
-    // This is necessary because multiple different build configurations will
-    // output different files here. Build cleaning only works when the files
-    // change within a build configuration.
+    // Deleting this directory is required or else the FlutterMacOS module
+    // cannot be found.
     if (targetDirectory.existsSync()) {
```

Comment:
Deleting this directory is still required, though I'm not sure why yet

Reply:
🤔